opendatahub-io/odh-dashboard
# [RHOAIENG-4825] Use execution input/output values as fallback for run node details

Closes: [RHOAIENG-4825](https://issues.redhat.com/browse/RHOAIENG-4825)

## Description
Use execution data for run node Input/Output parameter details.

<img width="1001" alt="Pasted Graphic 2" src="https://github.com/opendatahub-io/odh-dashboard/assets/96431149/a36b6df6-57c7-422e-bf47-a6d121ca187d">
(cc @yannnz)

## How to test
Navigate to a run with nodes that have Input/Output parameters associated with that run. Open the side panel by choosing one of those nodes, verify the contents of Input or Output parameters show actual values instead of just parameter types (e.g. STRING, although types are still a fallback, so if no execution data is found for a given task key, types will be shown).

## Test Impact
Unit tested with all possible types of input/output parameters with corresponding task execution values.

## Request review criteria:
Self checklist (all need to be checked):

If you have UI changes: 

After the PR is posted & before it merges:
